metadata.json

```diff
@@ -69,6 +69,6 @@
   ],
   "description": "These module provides an easy way to install and configure the clickhosue-server. It requires for 'xml-simple' gem",
   "pdk-version": "1.8.0",
-  "template-url": "https://github.com/puppetlabs/pdk-templates.git",
-  "template-ref": "heads/master-0-gbf720df"
-}
+  "template-url": "file:///home/felixoid/OPT/Felixoid/github/Felixoid/pdk-templates",
```

Comment:
Revert this change after the merge https://github.com/puppetlabs/pdk-templates/pull/171